packages/Python/lldbsuite/test/lang/swift/private_var/main.swift

```diff
@@ -13,7 +13,7 @@
 private var a = 1
 
 func doSomething(b: Int) {
-	a += strideof(b.dynamicType) //% self.expect("expr a", substrs=['Int', '= 1'])
+	a += MemoryLayout.of(b).stride //% self.expect("expr a", substrs=['Int', '= 1'])
```

Comment:
I asked Jim and Sean about this test, and they suggested that we use `a += b`, as long as the test continues to pass.